Fix assembly holes tolerances (#5)
AGH-CEAI/scene_props: README.md

### Insertion box
<img src="./docs/insertion_box_preview.png" alt="Insertion box (with holes) model preview" style="width:250px;"/>

> - It is hard to accurately cut the model into pieces with [O3DE's Convex Decomposition](https://www.docs.o3de.org/docs/learning-guide/tutorials/assets/physx-colliders/#convex-decomposition) for PhsyX engine (it is realized by [V-HACD library](https://github.com/kmammou/v-hacd)).

Reviewer: ```suggestion
- Accurately performing [convex decomposition](https://www.docs.o3de.org/docs/learning-guide/tutorials/assets/physx-colliders/#convex-decomposition) on this model in O3DE for the PhysX engine is challenging, as it relies on the [V-HACD library](https://github.com/kmammou/v-hacd).
```

### Single insertion box
<img src="./docs/single_insertion_box_preview.png" alt="Single insertion box (one hole) model preview" style="width:250px;"/>

### Peg - uniform prism
<img src="./docs/hexagon_peg.png" alt="Uniform hexagonal prism (peg)" style="width:250px;"/>

### Info
- These models are made with OpenSCAD software.
- By default, all metrics are unitless. Assume that **1 unit equals 1 mm**.
- Tolerances selected for holes are `5` and `10` mm.

> - Source `*.scad` files are in `src` directory.

Reviewer: ```suggestion
- Source `*.scad` files are located in the `src` directory.
```

## Requirements
```bash
sudo apt install openscad
```

---
## Development notes

> This project uses various tools for aiding the quality of the source code. Currently most of them are executed by the `pre-commit`. Please make sure to enable its hooks:

Reviewer: ```suggestion
This project uses various tools for aiding the quality of the source code. Currently most of them are executed by `pre-commit`. Please make sure to enable its hooks:
```

## Requirements
```bash

> sudo apt insatall openscad

Reviewer 2: Could you please fix a typo? (not a part of this PR, but noticed while screening)
```suggestion
sudo apt install openscad
```
